TelepresenceBot/core/src/main/java/com/novoda/tpbot/bot/BotTpService.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.tpbot.bot;
+
+import com.novoda.tpbot.Direction;
+import com.novoda.tpbot.Result;
+import com.novoda.tpbot.support.Observable;
+
+interface BotTpService {
```

Comment:
The plan is to use this in the follow-up presenters PR for the bot part of the app.